onboarding/app/controllers/customer_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CustomerController < ApplicationController
+  before_action :require_active_customer
```

Comment:
You need to exclude the create action from this before action.

Reply:
Actually, since the method that is being executed by the before action assumes that the customer is already in the database, you need to specify exactly which actions need to execute this.